### DISCUSSION TITLE
Issue with Turkish recognition "Altyazı M.K."

### DISCUSSION THREAD

**Discussion by oguzhansucuoglu** [openai/whisper#2412] [Q&A]
Sometimes randomly at the end of the text, Whipser adds "Altyazı M.K." for no reason. The voice is transcribed well and it finishes all of the audio, then this part is added at the end. How can I fix this?

**Comment by glangford** [comment]
FYI, probably related to this

* https://github.com/openai/whisper/discussions/928
